### ISSUE TITLE
Would be nice to have support field and field type annotations.

### ISSUE THREAD

**Issue by zolyfarkas** [mikaello/avrodoc-plus#13]
There is currently no way to see that.
One of the most important annotations is the "logicaType" one.

**Comment by mikaello** [comment]
I agree that this would have been a nice enhancement. I don't have time to work on such a feature at the moment, but maybe later. Or feel free to open a PR :-)

**Comment by mikaello** [comment]
Available in v1.5.0
